src/openvic-simulation/economy/ProductionType.cpp

```diff
@@ -4,23 +4,30 @@ using namespace OpenVic;
 using namespace OpenVic::NodeTools;
 
 Job::Job(
-	PopType const* const new_pop_type, effect_t new_effect_type, fixed_point_t new_effect_multiplier,
```

Comment:
What's wrong with the const here?

Reply:
It wasn't needed, just adds clutter, and the corresponding const had already been removed from the header file